extensions/wikia/Search/WikiaSearch.i18n.php

```diff
@@ -2,19 +2,21 @@
 $messages = array();
 
 $messages['en'] = array(
-	'wikiasearch2-page-title-with-query' => "Search results for '$1' - $2",
+	'wikiasearch2-page-title-with-query' => "Search results for '$1'",
```

Comment:
`wikiasearch2-page-title-with-query-crosswiki` is missing in this file ;)

Reply:
I forgot to remove `wikiasearch2-page-title-with-query-crosswiki` from code during cleanup, thanks for catching that